includes/forms/template.php

```diff
@@ -1139,50 +1207,40 @@ function give_get_register_fields( $form_id ) {
 			 */
 			do_action( 'give_register_account_fields_before', $form_id );
 			?>
-			<div id="give-user-login-wrap-<?php echo $form_id; ?>"
-				 class="form-row form-row-one-third form-row-first form-row-responsive">
-				<label for="give-user-login-<?php echo $form_id; ?>">
-					<?php esc_html_e( 'Username', 'give' ); ?>
-					<?php if ( give_logged_in_only( $form_id ) ) { ?>
-						<span class="give-required-indicator">*</span>
-					<?php } ?>
-					<?php echo Give()->tooltips->render_help( __( 'The username you will use to log into your account.', 'give' ) ); ?>
-				</label>
-
-				<input name="give_user_login" id="give-user-login-<?php echo $form_id; ?>" class="give-input"
-					   type="text"
-					   placeholder="<?php esc_attr_e( 'Username', 'give' ); ?>"<?php echo ( give_logged_in_only( $form_id ) ) ? ' required aria-required="true" ' : ''; ?>/>
-			</div>
 
-			<div id="give-user-pass-wrap-<?php echo $form_id; ?>"
-				 class="form-row form-row-one-third form-row-responsive">
-				<label for="give-user-pass-<?php echo $form_id; ?>">
-					<?php esc_html_e( 'Password', 'give' ); ?>
-					<?php if ( give_logged_in_only( $form_id ) ) { ?>
-						<span class="give-required-indicator">*</span>
-					<?php } ?>
-					<?php echo Give()->tooltips->render_help( __( 'The password used to access your account.', 'give' ) ); ?>
+			<div id="give-create-account-wrap-<?php echo $form_id; ?>" class="form-row form-row-first form-row-responsive">
+				<label for="give-create-account-<?php echo $form_id; ?>">
+					<?php
+					// Add attributes to checkbox, if Guest Checkout is disabled.
+					$is_guest_checkout = give_get_meta( $form_id,'_give_logged_in_only', true );
+					if( ! give_is_setting_enabled( $is_guest_checkout ) ) {
+						echo Give()->tooltips->render(
+							array(
+								'tag_content' => '<input type="checkbox" name="give_create_account" id="give-create-account-" class="give-input" checked disabled /><input type="hidden" name="give_create_account" value="on" />',
+								'label' => __( 'Registration is compulsory for this donation form as Guest Checkout is disabled.', 'give' ),
```

Comment:
@DevinWalker Do you want to change this message?

Reply:
Yes. It should be `Registration is required to donate.`